platform/authorized_api/users/public_v2.yaml

```diff
@@ -811,21 +790,21 @@ components:
             Forbidden:
               $ref: "#/components/examples/forbidden"
     id_mismatch:
-        description: Conflict
-        content:
-            application/vnd.api+json:
-                schema:
-                    allOf:
-                        -   $ref: "#/components/schemas/ErrorObject"
-                examples:
-                    Id mismatch:
-                      value:
-                        errors:
-                          - status: '409'
-                            code: error
-                            detail: "The resource object's id (985ed9d7-b652-47b8-b9cf-c17d62735261) does not match url's lookup id (8209367f-691b-475e-84c3-d1c69f46075b)"
-                            source:
-                               pointer: /data
+      description: Conflict
```

Comment:
Please update this pr to only contain changes to the specification, without any `x-api-*` attributes as this is related to the new portal. :pray: 

I also adjusted base branch to proper `18.32`.